docs/qualification/running-qualification.md

```diff
@@ -0,0 +1,61 @@
+# How to run qualification
+
+There are three ways to run the test and a couple of environment variables that a user should be aware of.
+
+??? tip "Suggested way of running"
+    Since this is a long test and bazel tends to be pretty heavy on the system it is proposed to run it on a devenv!
+    To do that you should follow the guide on how to create a devenv by idx and once its deployed you can:
+    ```bash
+    eval `ssh-agent -s`
+    ssh-add ~/.ssh/ssh_key_that_has_access_to_k8s_repo
+    ssh devenv
+    cd /to/root/of/ic/repo
+    ./ci/container/container-run.sh
+    ...
+    ```
+
+## Running using `ict`
+`ict` is a go-lang tool that was developed to help developers run tests with less friction in having to write long test names. To use `ict` one can spin up a new shell and:
+```bash
+cd /to/root/of/ic/repo
+./ci/container/container-run.sh
+ict test guest_os_qualification -- --test_timeout=7200 --keep_going
+```
+
+## Running using `bazel test`
+Spin up a new shell and:
+```bash
+cd /to/root/of/ic/repo
+./ci/container/container-run.sh
+bazel test //rs/tests/dre:guest_os_qualification --config=systest --cache_test_results=no --test_env=IC_DASHBOARDS_DIR=/path/to/k8s_repo/bases/apps/ic-dashboards --sandbox_add_mount_pair=/path/to/k8s_repo/bases/apps/ic-dashboards --test_timeout=7200 --keep_going
+```
+
+## Environment variables
+- `OLD_VERSION`: specifies the starting version for a testnet. If its not specified it will default to the version specified in `tests/mainnet_revision.json`.
```

Comment:
```suggestion
- `OLD_VERSION`: specifies the starting version for a testnet. If it's not specified it will default to the version specified in `tests/mainnet_revision.json`.
```
https://www.merriam-webster.com/grammar/when-to-use-its-vs-its